e2e_playwright/hostframe_app_test.py

```diff
@@ -56,7 +56,7 @@ def fulfill_host_config_request(route: Route):
     toolbar = page.get_by_test_id("toolbar")
     expect(toolbar).to_have_count(1)
     toolbar_buttons = toolbar.get_by_role("button")
-    expect(toolbar_buttons).to_have_count(12)
+    expect(toolbar_buttons).to_have_count(13)
```

Comment:
New to these tests -- this change is because you added the `file_uploader` to the `hostframe_app.py`? Wouldn't that be inside the iframe?

Reply:
This is for the parent frame. 
I added the `Set file upload config` button to the `hostframe.html` to send config and test the flow e2e. 

https://github.com/streamlit/streamlit/pull/9727/files#diff-fc4ee9383485b28b70e31e605b62bc44a614d20b5486fbd5c1be78d484d4ed97R283